ext/opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -292,7 +300,9 @@ def _extract_events(events: Sequence[Event]) -> ProtoSpan.TimeEvents:
 
 
 def _extract_attributes(
-    attrs: types.Attributes, num_attrs_limit: int
+    attrs: types.Attributes,
+    num_attrs_limit: int,
+    span_attributes: bool = False,
```

Comment:
What does `span_attributes` mean? Looking at the code I'd call this `add_agent_attr` or similar.

Reply:
I intended it to be a flag saying it was called to actually retreive the span_attributes, vs attributes from a span's events. add_agent_attr seems clearer